src/huak/config/pyproject/project/mod.rs

```diff
@@ -41,6 +42,7 @@ impl Default for Project {
             authors: vec![],
             dependencies: vec![],
             optional_dependencies: None,
+            scripts: None,
```

Comment:
If we don't default to Library, wouldn't we need the Project to default with a script? I do think we should default to a Library, so we may not need to make a change here, but I just thought 'd point that out.

Reply:
good point.